k-distribution/tests/regression-new/rat/defined/java/4.a.out

```diff
@@ -1,5 +1,5 @@
 <T>
   <k>
-    < 29 , 18 > ;  < 1 , 18 > ;  < 35 , 54 > ;  < 15 , 14 > ;  < 52 , 9 > ;  < 38 , 9 > ;  < 35 , 9 > ;  < 45 , 7 > ;  < 47 , 6 > ;  < -37 , 6 > ;  < 35 , 6 > ;  < 5 , 42 > ;  12 ;  -2 ;  35 ;  < 5 , 7 > ;  .Pgm
+    29 /Rat 18 ;  1 /Rat 18 ;  35 /Rat 54 ;  15 /Rat 14 ;  52 /Rat 9 ;  38 /Rat 9 ;  35 /Rat 9 ;  45 /Rat 7 ;  47 /Rat 6 ;  -37 /Rat 6 ;  35 /Rat 6 ;  5 /Rat 42 ;  12 ;  -2 ;  35 ;  5 /Rat 7 ;  .Pgm
```

Comment:
Once again, why do we not simplify down to the constructor anymore?

Reply:
This is similar to the other one, but a little different. We made the internal constructor private, so the rule that was present in the K definition that rewrote the internal construtor into the constructor seen here in the output file was no longer able to be written. I just deleted that rule since it wasn't really necessary, and as a result, the output contains just the internal constructor, which gets written via the format attribute as /Rat.